source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -432,13 +446,102 @@ void SplitKeysSumResultRequest::onChildResponse(Common::Redis::RespValuePtr&& va
   }
 }
 
+SplitRequestPtr TransactionRequest::create(Router& router,
+                                           Common::Redis::RespValuePtr&& incoming_request,
+                                           SplitCallbacks& callbacks, CommandStats& command_stats,
+                                           TimeSource& time_source, bool delay_command_latency) {
+  Common::Redis::Client::Transaction& transaction = callbacks.transaction();
+  std::string to_lower_string = absl::AsciiStrToLower(incoming_request->asArray()[0].asString());
```

Comment:
nit: might be better to change variable name "to_lower_string" to "command_name". I don't think "to_lower_string" in existing code is straightforward to read.

Reply:
Fixed